packages/core/src/external/carequality/iti-39-parsing.ts

```diff
@@ -15,7 +15,8 @@ const documentData: { [key: string]: string } = {
  * @param xml - The XML string to be parsed.
  * @returns A promise that resolves to an array containing the signature, documentId, and homeCommunityID extracted from the XML.
  */
-async function parseXmlString(xml: string): Promise<[string, string, string]> {
+async function parseXmlString(xml: string, header: string): Promise<[string, string, string]> {
+  xml = parseMtomResponse(xml, header);
   xml = cleanXml(xml);
```

Comment:
Unless we're dealing with very large objects/contents, I recommend we don't modify the input parameters.